pkgs/press/src/constants.ts

```diff
@@ -5,4 +5,4 @@ export const REGION = 'us-west-1'
 export const SENTRY_CLI = resolve(__dirname, '../node_modules/.bin/sentry-cli')
 export const TAG_BRANCH_PREFIX = 'branch-'
 export const TAG_COMMIT_PREFIX = 'commit-'
-export const TAG_VERSION_PREFIX = process.env.TAG_VERSION_PREFIX || 'v'
+export const TAG_VERSION_PREFIX = 'version-'
```

Comment:
Why did you revert this?

Reply:
so that it can handle prefixed versioned already in the ECR repo. it  does it when validating current tag against existing ones (new semver < existing semver's)

Reply:
Got it. Was wondering if that was going to be an issue with your last PR.